src/contrib/cluster/Akka.Cluster.Tools.Tests/Singleton/ClusterSingletonConfigSpec.cs

```diff
@@ -5,6 +5,8 @@
 // </copyright>
 //-----------------------------------------------------------------------
 
+#region ClusterSingletonConfigSpec
```

Comment:
Don't want to include the _entire_ test here - only need the constructor.

Use the `//<{name}>` syntax to target a small area of code, rather than a `#region` 

See: https://getakka.net/community/contributing/documentation-guidelines.html#code-samples-must-use-code-references